### PR TITLE
Email Popup UI Overhaul

### PR DESCRIPTION
**Pull request by roopalkondepudi**
There are a few nit-picky things I still need to figure out how to do - specifically, changing the default text.

See below for 'completed' product.

Screen Shot 2020-09-19 at 1 01 58 AM

### CODE REVIEW

**Comment by roopalkondepudi** [comment]
I realized that in the process I somehow broke the send functionality... that will be fixed ASAP

File: letterOfRecGenerator/views/pages/recommender-dashboard.ejs [hunk 1]
```diff
@@ -37,29 +37,32 @@
...
-                    <textarea id="body-text" name="body_text" value = "<%= body%>" ><%= body %></textarea>
-                </div>
-                <input class="input-submit" type="submit" value="Send">
-            </form>
+                    <br>
+                    <div class="email-container">
+                        <div class="input-container">
+                            <p class="subject-text"> Subject: </p>
+                            <input class="input-text" id="subject" type="text" name="subject_text" value="<%= subject%>" >
+                            <!-- <button id = "Edit"> Edit </button> -->
```
Comment (nolan1299) [inline letterOfRecGenerator/views/pages/recommender-dashboard.ejs]: comment D:

File: letterOfRecGenerator/public/css/recommender-dashboard.css [hunk 2]
```diff
@@ -150,3 +150,15 @@ img.clickable:hover {
...
+
+#template-select{
+    color: gray;
+}
+
+.template-select-option{
+    color: black;
+}
+
+/* #template-select-placeholder #template-select-placeholder{
```
Comment (nolan1299) [inline letterOfRecGenerator/public/css/recommender-dashboard.css]: D:

**Review by nolan1299 — CHANGES_REQUESTED** [review]
just some leftover  debugging comments

**Comment by nolan1299** [comment]
also prob a good idea to squash some of the commits to keep login2 fresh and clean

**Review by nolan1299 — APPROVED** [review]

**Pull request merged**